theseus_gui/src/components/ui/InstallConfirmModal.vue

```diff
@@ -0,0 +1,53 @@
+<script setup>
+import { Button, Modal, XIcon, DownloadIcon } from 'omorphia'
+import { useRouter } from 'vue-router'
+import { install as pack_install } from '@/helpers/pack'
+import { ref } from 'vue'
+
+const router = useRouter()
+
+const version = ref('')
+const confirmModal = ref(null)
+
+defineExpose({
+  show: (id) => {
+    version.value = id
+    confirmModal.value.show()
+  },
+})
+
+async function install() {
+  let id = await pack_install(version.value)
+  await router.push({ path: `/instance/${encodeURIComponent(id)}` })
+}
+</script>
+
+<template>
+  <Modal header="Are you sure?" ref="confirmModal">
+    <div class="modal-body">
+      <p>
+        This project is already installed on your system. Are you sure you want to install it again?
+      </p>
+      <div class="button-group">
+        <Button @click="() => $refs.confirmModal.hide()"><XIcon />Cancel</Button>
+        <Button @click="install()" color="primary"><DownloadIcon /> Install</Button>
```

Comment:
would also make the modal close if the install button is clicked (as it will be replaced with the whole progress bar)